src/app/components/TransactionsTable/index.tsx

```diff
@@ -30,145 +54,215 @@ export default function TransactionsTable({
     <p className="text-gray-500 dark:text-neutral-400">{noResultMsg}</p>
   ) : (
     <>
-      <div className="shadow overflow-hidden rounded-lg bg-white divide-y divide-gray-200 dark:divide-white/10 dark:bg-surface-02dp">
-        {transactions?.map((tx) => (
-          <div key={tx.id} className="px-3 py-2">
-            <Disclosure>
-              {({ open }) => (
-                <>
-                  <div className="flex">
-                    <div className="overflow-hidden mr-3">
-                      <div
-                        className="
-                  text-sm font-medium text-gray-900 truncate dark:text-white"
-                      >
-                        <p className="truncate">
-                          {tx.publisherLink && tx.title ? (
-                            <a
-                              target="_blank"
-                              href={tx.publisherLink}
-                              rel="noopener noreferrer"
-                            >
-                              {tx.title}
-                            </a>
-                          ) : (
-                            tx.title || tx.boostagram?.message || "\u00A0"
-                          )}
-                        </p>
-                      </div>
-                      <p className="text-xs text-gray-600 dark:text-neutral-400">
-                        {tx.date}
+      <div className="overflow-hidden">
+        {transactions?.map((tx) => {
+          const type = getTransactionType(tx);
+
+          return (
+            <div
+              key={tx.id}
+              className="px-3 py-2 hover:bg-gray-100 dark:hover:bg-surface-02dp cursor-pointer rounded-md"
+              onClick={() => openDetails(tx)}
+            >
+              <div className="flex gap-3">
+                <div className="flex items-center">
+                  {type == "outgoing" ? (
+                    <div className="flex justify-center items-center bg-orange-100 rounded-full w-8 h-8">
+                      <ArrowUpIcon className="w-6 h-6 text-orange-400" />
+                    </div>
+                  ) : (
+                    <div className="flex justify-center items-center bg-green-100 rounded-full w-8 h-8">
+                      <ArrowDownIcon className="w-6 h-6 text-green-400" />
+                    </div>
+                  )}
+                </div>
+                <div className="overflow-hidden mr-3">
+                  <div
+                    className="
+      text-sm font-medium text-black truncate dark:text-white"
+                  >
+                    <p className="truncate">
+                      {tx.publisherLink && tx.title ? (
```

Comment:
it's strange and not intuitive how the title link works. What about we always expand to show more details on click, and then have an explicit link inside there instead?

Reply:
resolved